ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public protocol PersonaCellProtocol {
+    @objc var persona: MSFAvatarState? { get set }
+    @objc var titleTrailingAccessoryView: UIView? { get set }
+    @objc var subtitleTrailingAccessoryView: UIView? { get set }
+    @objc var onTapAction: (() -> Void)? { get set }
+}
+
+@objc public class MSFPersonaCellState: MSFListCellState, PersonaCellProtocol {
+    @objc @Published public var persona: MSFAvatarState?
+}
+
+struct PersonaView: View {
+    @ObservedObject var state: MSFPersonaCellState
+    @ObservedObject var tokens: MSFListCellTokens
+
+    init(state: MSFPersonaCellState, windowProvider: FluentUIWindowProvider?) {
+        self.state = state
+        self.tokens = MSFListCellTokens(cellLeadingViewSize: .xlarge, style: .persona)
+        self.tokens.windowProvider = windowProvider
+    }
+
+    var body: some View {
+        MSFListCellView(state: getCellState(), style: .persona, windowProvider: tokens.windowProvider)
+    }
+
+    private func getCellState() -> MSFListCellState {
+        let cellState = MSFListCellState()
+        cellState.leadingView = createAvatar().view
```

Comment:
Can we try to provide a SwifUI View as a leading view in the cell (while supporting the UIView)?
Then we don't need the UIView adapter here since this is a "SwiftUI x SwiftUI" interaction.

Reply:
The only way I can think of doing this is creating another property type, or making the leadingView type an enum. This is because I don't think AnyView/View (in SwiftUI) can be marked with an @ objc

Reply:
SwiftUI types can't be exposed in ObjC indeed. But we don't necessarily need it to be exposed there.
We can have a property like the sample below in the state and give priority to it, falling back to the UIView property.

`@Published public var leadingView: AnyView?`

Better yet, you can have the setter of the UIView property wrap the UIView passed in a representable (UIViewAdapter) type and set it in the AnyView property?